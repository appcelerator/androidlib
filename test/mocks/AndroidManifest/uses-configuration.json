{
	"uses-configuration": [
		{
			"android:reqFiveWayNav": true,
			"android:reqKeyboardType": "qwerty",
			"android:reqTouchScreen": "finger"
		},
		{
			"android:reqFiveWayNav": true,
			"android:reqKeyboardType": "twelvekey",
			"android:reqTouchScreen": "finger"
		}
	]
}
