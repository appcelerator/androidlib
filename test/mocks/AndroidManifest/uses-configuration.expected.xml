<?xml version="1.0" encoding="utf-8"?>
<manifest>
  <uses-configuration android:reqFiveWayNav="true" android:reqKeyboardType="qwerty" android:reqTouchScreen="finger"/>
  <uses-configuration android:reqFiveWayNav="true" android:reqKeyboardType="twelvekey" android:reqTouchScreen="finger"/>
</manifest>
